setup.py

```diff
@@ -121,7 +113,6 @@ def read(*names):
         "Topic :: Software Development :: Libraries :: Python Modules",
         "Topic :: Software Development :: Testing",
     ],
-    namespace_packages=['zope'],
```

Comment:
This change goes too far by switching to implicit namespaces (PEP-420) – It would require all other 117 packages of this namespace to switch over at the same time.
See https://github.com/zopefoundation/meta/issues/194 where we track our progress regarding this PEP.

If the issue cannot be solved without the switch to PEP-420, it has to wait until we have everything in place to to the swtich for all packages.

Reply:
> This change goes too far by switching to implicit namespaces (PEP-420) – It would require all other 117 packages of this namespace to switch over at the same time.

I have to admit this is my first time interacting with namespace packages, so I might be out on a limb here, but this does not seem to be the case:

```sh
$ python -m venv .venv && source ./.venv/bin/activate
(.venv) $ pip install zope.interface
...
Successfully installed setuptools-73.0.1 zope.interface-7.0.1
(.venv) $ pip install dist/zope.testrunner-6.6.dev0-py3-none-any.whl
...
Successfully installed zope.exceptions-5.1 zope.testrunner-6.6.dev0
(.venv) $ python -c "import zope.interface; import zope.testrunner;"
```

I.e. testrunner built from this branch can be installed and imported without issues alongside `zope.interface`, which uses an explicit namespace declaration. Maybe I'm missing something?

Reply:
It's a complicated matter: It sometimes depends on import order and installing packages using `zc.buildout` (as done often in the Zope community) also seems to play a role. So our plans are to do the switch for all packages together to circumvent the problems … but this will require time and coordination effort.

Reply:
Aha, I see. I've removed the migration to implicitly declared namespace in https://github.com/zopefoundation/zope.testrunner/pull/179/commits/e9b96914fbabace1b66b842e7b0fb13695026b7b. This leaves the exclusion of the tests from the wheel, is that something you could consider merging?